core/jobs/batch_jobs/audit_exploration_jobs.py

```diff
@@ -0,0 +1,1479 @@
+                        file_with_value = rte_component['customization_args'][
+                            'filepath-with-value']
+                    except Exception:
+                        file_with_value = 'Not found'
+
+                    # Validates filepath extension is svg.
+                    if (
+                        file_with_value != 'Not found' and
+                        len(file_with_value) > 0 and
+                        file_with_value[-3:] != 'svg'
+                    ):
+                        rte_image_errors.append(
+                            f'State - {state_name} Image tag filepath value '
+                            f'at index {idx} does not have svg extension '
+                            f'having value {file_with_value}.'
+                        )
+
+            states_with_values.append(
+                {
+                    'state_name': state_name,
+                    'rte_image_errors': rte_image_errors
+                }
+            )
+        return states_with_values
+
+    @staticmethod
+    def filter_invalid_rte_video_tag(
+        states_dict: Dict[str, state_domain.State]
+    ) -> List[Dict[str, List[Dict[str, List[str]]]]]:
+        """Returns the errored state values, validates
+            - Start value is before end value
+
+        Args:
+            states_dict: dict[str, State]. The dictionary containing
+                state name as key and State object as value.
+
+        Returns:
+            states_with_values: list[dict]. The list of dictionaries
+            containing the errored values.
+        """
+        states_with_values = []
+
+        for state_name, state in states_dict.items():
+            rte_video_errors = []
+            rte_components = ExpStateAuditChecksJob._get_rte_components(
+                state.content.html)
+            for idx, rte_component in enumerate(rte_components):
+                if rte_component['id'] == 'oppia-noninteractive-video':
+                    try:
+                        start_value = (
+                            rte_component['customization_args']
+                            ['start-with-value'])
+                        if start_value == '':
+                            start_value = '0'
+                    except Exception:
+                        start_value = 'Not found'
+
+                    try:
+                        end_value = (
+                            rte_component['customization_args']
+                            ['end-with-value'])
+                        if end_value == '':
+                            end_value = '0'
+                    except Exception:
+                        end_value = 'Not found'
+
+                    if (
+                        (
+                            start_value != 'Not found' and
+                            end_value != 'Not found'
+                        )
+                        and int(start_value) > int(end_value)
+                    ):
```

Comment:
`start-with-value` and `end-with-value` don't need to be integers. Better to do:
`float(start_value) > float(end_value)`.

Reply:
Done